client/rpc-api/src/system/mod.rs

```diff
@@ -25,8 +25,9 @@ use futures::{future::BoxFuture, compat::Compat};
 
 use self::error::Result as SystemResult;
 
-pub use self::helpers::{Properties, SystemInfo, Health, PeerInfo, NodeRole};
+pub use self::helpers::{SystemInfo, Health, PeerInfo, NodeRole};
 pub use self::gen_client::Client as SystemClient;
+use sp_chain_spec::Properties;
```

Comment:
Might be worth to either consistently import both `Properties` and `ChainType` or use full path in both places.